src/AssemblySharedInfoGenerator/AssemblySharedInfo.cs

```diff
@@ -64,4 +64,4 @@
 // You can specify all the values or you can default the Build and Revision Numbers
 // by using the '*' as shown below:
 // [assembly: AssemblyVersion("1.0.*")]
-[assembly: AssemblyFileVersion("2.19.0.4737")]
+[assembly: AssemblyFileVersion("2.19.0.5077")]
```

Comment:
please don't commit this file in general - it just creates conflicts for others prs.

Reply:
Ok, I'll avoid commiting this file next time